mybatis-spring-boot-autoconfigure/src/main/java/org/mybatis/spring/boot/autoconfigure/MybatisProperties.java

```diff
@@ -409,6 +409,11 @@ public static class CoreConfiguration {
      */
     private Properties variables;
 
+    /**
+     * Specifies the databaseId
```

Comment:
please JavaDoc end with '.' and I prefer description such as `"Specifies the database identify value for switching query to use."`.

Reply:
> please JavaDoc end with '.' and I prefer description such as `"Specifies the database identify value for switching query to use."`.

Thank you for your suggestion, I have adjusted it